SpaDES4Dummies.Rmd

```diff
@@ -10,116 +10,142 @@ output: html_document
+You will notice that `newModule` has created a module folder (`speciesAbundance`) inside */m* that contains both the module *.R* script template, as well as the documentation template (the *.Rmd* file). Although we will not be discussing the *.Rmd* file, please bear in mind that this is a **fundamental** part of creating a reproducible and transparent module - check out the [Guide to Reproducible Code in Ecology and Evolution](http://www.britishecologicalsociety.org/wp-content/uploads/2017/12/guide-to-reproducible-code.pdf) from the British Ecological Society). The documentation should contain not only the description of the module, but also some simple examples of what it can do.
 
-`newModule` also created the folder */data* where data necessary to the module should be put in, and the folder */tests* that may contain testing scripts. We won't be using neither of them in this example. 
+`newModule` also created the folder */data* where data necessary to the module should be put in, and the folder */tests* that may contain testing scripts. We won't be using either of them in this example. 
 
 **/!\\ Attention /!\\** : `newModule` should only be run once, or separately in the console, otherwise you may lose your module edits by re-running it again - this is why I wrapped it in an `if` statement (smart eh?).
 
+Now go ahead, open the *speciesAbundance.R* script and check it out a little bit.
+
+***
+
+#### SPECIES ABUNDANCE (`speciesAbundance`) Module
 
-Now go ahead and open the *speciesAbundance.R* script. I know, I know... It seems confusing and there's a lot of stuff in that template! 
+I know, I know... It seems confusing and there's a lot of stuff in that template! But I will describe it step by step. We will follow a different order from the `speciesAbundance.R` script file. We divide the Module script into 4 chunks: 
 
-The first thing you need to know is that **you don't need to run** any of the code inside a module *.R* script. The function `simInit` will do it for you when it sets up the simulation (you'll see this later). But before that, let's go throught the module *.R* script together.
+**1. [Defining the Module](#anchor1):** this is where you **define** your module (or the module's metadata) (e.g. module author(s), time units, basic parameters, general inputs and outputs, etc.);  
+**2. [Event functions](#anchor2):** these are the "actions" (or events) that will happen in your module (i.e. species reproduction, plotting, saving parameters) - simply put, **WHAT** the module will do;  
+**3. [Scheduling Events](#anchor3):** this is where SpaDES schedules when each event is going to happen - in which order (e.g. during the simulation, when will SpaDES plot a graph) - simply put, **WHEN** the module will do;  
```

Comment:
instead:
**3. [Scheduling Events](#anchor3):** this is how SpaDES schedules when (...)